services/file-storage-and-transfer/brown-storage.md

```diff
@@ -3,9 +3,233 @@ title: Brown Storage Options
+        </li>
+        <li>Limitations: Not accessible on all campus networks.</li>
+        <li>Rate: $100/TB/Year when storing above free/grant allocations</li>
+        <li>
+            More info: 
+            <a href="https://ccv.brown.edu/services/computing#high-performance-computing-(oscar)">Documentation</a> | 
+            <a href="https://brown.co1.qualtrics.com/jfe/form/SV_0GtBE8kWJpmeG4B">Request these services</a>
+        </li>
+    </ul>
+</details>
+
+<details>
+    <summary>Stronghold</summary>
+    <p>
+        <a href="https://it.brown.edu/services/stronghold-research-environment-data-compliance">Stronghold</a> 
+        is a secure computing and storage environment that enables Brown researchers to analyze sensitive data 
+        while complying with regulatory or contractual requirements.
+    </p>
+    <ul>
+        <li>Best for: Storing data with data usage agreements, FISMA, etc.</li>
+        <li>Rate: $100/TB/Year when storing above free/grant allocations</li>
+        <li>More info: <a href="https://www.brown.edu/cis/forms/Stronghold/shold.php">Request this service</a></li>
+    </ul>
+</details>
+
+<details>
+    <summary>Hibernate</summary>
+    <p>
+        <a href="https://docs.ccv.brown.edu/hibernate/">Hibernate</a> is a secure, reliable,
+        research data archive solution. Hibernate is a Brown OIT archival service for the 
+        research community to migrate inactive data off active Network-attached storage (NAS)
+        platforms onto a lower cost, long-term retention environment.
+    </p>
+    <ul>
+        <li>
+            Hibernate leverages <a href="https://docs.ccv.brown.edu/starfish/">StarFish</a> 
+            an application that provides a metadata and rules-based 
+            management framework for large file systems. StarFish makes storage tiering easy: 
+            moving data, reporting, zones.
+        </li>
+    </ul>
+</details>
+
+<details>
+    <summary>Lab Archives</summary>
+    <p>
+        LabArchives is a cloud-based electronic lab notebook that can be used by researchers, 
+        instructors, and students for input and organization of laboratory data, information 
+        sharing, and collaboration, and for saving historical versions of files. It is appropriate 
+        for use in a wide variety of laboratories, including biological sciences, chemistry and 
+        physical sciences, and engineering, among others.
+    </p>
+    <p>
+        LabArchives at Brown provides unlimited storage space. The current size limit per file is 4GB.
+    </p>
+    <p>
+        LabArchives at Brown is not approved for storing files containing Personally Identifiable 
+        Information (PII), Protected Health Information (PHI), or Brown Restricted Information.
+    </p>
+    <ul>
+        <li>More info: <a href="https://library.brown.edu/info/labarchives/">Documentation</a></li>
+    </ul>
+</details>
+
+<details>
+    <summary>Brown Digital Library</summary>
```

Comment:
Shouldn't this be `Brown Digital Repository`?

Reply:
```suggestion
    <summary>Brown Digital Repository</summary>
```